packages/at_client/lib/src/service/sync_service_impl.dart

```diff
@@ -375,9 +375,10 @@ class SyncServiceImpl implements SyncService, AtSignChangeListener {
       List<CommitEntry> uncommittedEntries) async {
     // Iterates until serverCommitId is greater than localCommitId are equal.
     List<KeyInfo> keyInfoList = [];
-    while (serverCommitId > localCommitId) {
+    int lastReceivedServerCommitId = localCommitId;
+    while (serverCommitId > lastReceivedServerCommitId) {
       var syncBuilder = SyncVerbBuilder()
-        ..commitId = localCommitId
+        ..commitId = lastReceivedServerCommitId
```

Comment:
Pls document what should be the right value of a commitId need to be

Reply:
done